README.md

```diff
@@ -81,6 +81,25 @@ WebPack will search your `node_modules` folder by default, so if you include `ab
 
 If that doesn't work for some reason (e.g. you are using a custom webpack target), you can reference the `ably.js` static file directly: `require('ably/build/ably.js');` (or `import * as Ably from 'ably/build/ably.js'` for typescript / ES6 modules).
 
+#### Modular (tree-shakable) variant
+
+Aimed at those who are concerned about their app’s bundle size, the modular variant of the library allows you to create a client which has only the functionality that you choose. Unused functionality can then be tree-shaken by your module bundler.
+
+To use this variant of the library, import `ably/modules`. It exposes a `BaseRealtime` export, as well as various modules that provide functionality, such as `Rest`, `RealtimePresence`, etc. To create a client, choose which modules you wish to use and pass them to the constructor:
```

Comment:
I don't like "import `ably/modules`" as a clause, it feels a bit imprecise to me? I tried to rewrite the paragraph to avoid it, but I still don't love it.

```suggestion
To use this variant of the library, import the `BaseRealtime` class from `ably/modules`, along with the additional modules you wish to use. Various modules such as `Rest`, `RealtimePresence`, etc. are available. To create a client, simply pass the additional modules to the `BaseRealtime` constructor as shown in the example below:
```

Reply:
I've tweaked the wording based on your suggestion; see what you think.